Source/WUtils.swift

```diff
@@ -23,4 +23,47 @@ public enum xAlignment {
 }
 public enum yAlignment {
     case Top, Center, Bottom
-}
\ No newline at end of file
+}
+
+public class WUtils {
```

Comment:
Are these only used for testing right now? If so we should move it to `WTestUtils`.